parts/configure-swarmmode-cluster.sh

```diff
@@ -154,7 +164,7 @@ updateDockerDaemonOptions()
     # also have it bind to the unix socket at /var/run/docker.sock
     sudo bash -c 'echo "[Service]
     ExecStart=
-    ExecStart=/usr/bin/docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock
+    ExecStart=/usr/bin/docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock -H fd:// --experimental=true --metrics-addr=0.0.0.0:4999

```

Comment:
These changes suggested above not complete and don't work. Should I open a feature request for passing Orchestrator options?